doc/GETSTARTED.md

```diff
@@ -0,0 +1,77 @@
+# Get Started
+
+- Skill requirements
+
+  - Operating system
+
+    We will need a unix-based operating system to run Elektra (Linux, BSD, macOS) but it's also possible to use Windows which is supported but not tested much. If you are new to the world of unix we recommend you to use one of the common linux distributions like [Ubuntu](https://ubuntu.com/download/desktop#download), [Linux Mint](https://www.linuxmint.com/download.php), [openSuse](https://software.opensuse.org/distributions/leap) or [Fedora](https://getfedora.org/en/).
+
+  - Using command-line interface and commands
+
+    The easiest way to compile, install and use Elektra is by using the terminal. We will introduce the basic commands which you will need to run Elektra for the very first time.
+
+  - Basic knowledge about git
+
+    Dont panic! [GIT](https://git-scm.com/) is a distributed version constrol system to track changes of the source code in a project. We will use a single command of GIT to get
+    the source code of Elektra.
+
+  - Basic knowledge about make/cmake
+
+    Never heard about [make](https://www.gnu.org/software/make/) or [cmake](https://cmake.org/), it’s okey! We will use this build-management tools to compile, link dependencies etc. to build a executable program from the source code.
```

Comment:
If someone really did not hear about make or cmake he will also have troubles understanding "link dependencies" (Depite I know both tools for example I do not exactly know what happens on dependency linkage).

Reply:
If you will have troubles with cmake depends on what you want to do. If you only write a simple plugin, there should be no problem.